src/ai/defaultai.cc

```diff
@@ -6783,7 +6780,8 @@ void DefaultAI::review_wares_targets(const Time& gametime) {
 	tribe_ = &player_->tribe();
 
 	// to avoid floats real multiplier is multiplier/10
-	const uint16_t multiplier = std::max<uint16_t>((productionsites.size() + num_ports * 5) / 5, 10);
+	const uint16_t multiplier = std::max<uint16_t>(
+	   static_cast<uint16_t>((productionsites.size()) + num_ports * 5) / 5, 10);  // NOLINT
```

Comment:
```suggestion
	   (static_cast<uint16_t>(productionsites.size()) + num_ports * 5) / 5, 10);  // NOLINT
```
the problem here were the wrong placed brackets. May we have a try if this would work